!Command
positional: []
named: []
command:
- quality_scores_plot.py
parent:
subcommands: []
help_flag:
usage_flag:
version_flag:
help_text: "Traceback (most recent call last):\n  File \"/tmp/tmpzblxglq9/bin/quality_scores_plot.py\"\
  , line 4, in <module>\n    __import__('pkg_resources').run_script('qiime==1.9.1',\
  \ 'quality_scores_plot.py')\n  File \"/tmp/tmpzblxglq9/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3251, in <module>\n    @_call_aside\n  File \"/tmp/tmpzblxglq9/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3235, in _call_aside\n    f(*args, **kwargs)\n  File \"/tmp/tmpzblxglq9/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 3264, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n\
  \  File \"/tmp/tmpzblxglq9/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 583, in _build_master\n    ws.require(__requires__)\n  File \"/tmp/tmpzblxglq9/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 900, in require\n    needed = self.resolve(parse_requirements(requirements))\n\
  \  File \"/tmp/tmpzblxglq9/lib/python2.7/site-packages/pkg_resources/__init__.py\"\
  , line 786, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound:\
  \ The 'backports.functools-lru-cache>=1.2.1; python_version < \"3.2\"' distribution\
  \ was not found and is required by wcwidth\n"
generated_using:
- --help
